sql/moz-fx-data-shared-prod/firefox_accounts/fxa_content_auth_stdout_events/view.sql

```diff
@@ -20,147 +20,54 @@
 ---  'auth',
 --   'stdout'
 --- )
+--------
+-- In the meantime, this view points to the fxa_all_events view
+-- while maintaining filtering rules to only include certain events for backward compatibility.
 -----
 CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.firefox_accounts.fxa_content_auth_stdout_events`
 AS
-WITH content AS (
-  SELECT
-    jsonPayload.logger,
-    jsonPayload.fields.event_type,
-    jsonPayload.fields.app_version,
-    jsonPayload.fields.os_name,
-    jsonPayload.fields.os_version,
-    jsonPayload.fields.country,
-    CAST(NULL AS STRING) AS country_code,
-    jsonPayload.fields.language,
-    jsonPayload.fields.user_id,
-    jsonPayload.fields.device_id,
-    jsonPayload.fields.user_properties,
-    jsonPayload.fields.event_properties,
-    `timestamp`,
-    receiveTimestamp,
-    TIMESTAMP_MILLIS(CAST(jsonPayload.fields.time AS INT64)) AS event_time,
-  FROM
-    `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_content_events_v1`
-),
-  --
-auth AS (
-  SELECT
-    jsonPayload.logger,
-    jsonPayload.fields.event_type,
-    jsonPayload.fields.app_version,
-    jsonPayload.fields.os_name,
-    jsonPayload.fields.os_version,
-    jsonPayload.fields.country,
-    JSON_VALUE(jsonPayload.fields.event_properties, "$.country_code") AS country_code,
-    jsonPayload.fields.language,
-    jsonPayload.fields.user_id,
-    jsonPayload.fields.device_id,
-    jsonPayload.fields.user_properties,
-    jsonPayload.fields.event_properties,
-    `timestamp`,
-    receiveTimestamp,
-    TIMESTAMP_MILLIS(CAST(jsonPayload.fields.time AS INT64)) AS event_time,
-  FROM
-    `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_auth_events_v1`
-),
-  --
-stdout AS (
-  SELECT
-    jsonPayload.logger,
-    jsonPayload.fields.event_type,
-    jsonPayload.fields.app_version,
-    jsonPayload.fields.os_name,
-    jsonPayload.fields.os_version,
-    CAST(NULL AS STRING) AS country,
-    jsonPayload.fields.country_code,
-    jsonPayload.fields.language,
-    jsonPayload.fields.user_id,
-    jsonPayload.fields.device_id,
-    jsonPayload.fields.user_properties,
-    jsonPayload.fields.event_properties,
-    `timestamp`,
-    receiveTimestamp,
-    TIMESTAMP_MILLIS(CAST(jsonPayload.fields.time AS INT64)) AS event_time,
-  FROM
-    `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_stdout_events_v1`
-),
-  --
-unioned AS (
-  SELECT
-    *
-  FROM
-    auth
-  UNION ALL
-  SELECT
-    *
-  FROM
-    content
-  UNION ALL
-  SELECT
-    *
-  FROM
-    stdout
-)
-  --
 SELECT
```

Comment:
This removed 13 columns from the view, which should be added back.
```suggestion
SELECT
  logger,
  event_type,
  app_version,
  os_name,
  os_version,
  country,
  country_code,
  `language`,
  user_id,
  device_id,
  `timestamp`,
  receiveTimestamp,
  event_time,
```

Reply:
thanks for catching this. I thought I had them in there. I guess I missed them in this view. Opened https://github.com/mozilla/bigquery-etl/pull/4307 to re-add them.